### ISSUE TITLE
Cannot run this player on any android browser.

### ISSUE THREAD

**Issue by yang3kui** [hawksley/eleVR-Web-Player#17]
When using firefox, the orentation sensor  works in a weird manner,  the rotation of the image dosenot conform to the rotation of the phone,  in both Web-Player and Picture-player. 

When using chrome, everything is fine with Picture-player, however, the audio plays but no visible image shows in the Web-Player.

Can anybody help?

**Comment by hawksley** [comment]
Duplicate of issues 
https://github.com/hawksley/eleVR-Web-Player/issues/12
and
https://github.com/hawksley/eleVR-Web-Player/issues/11

Both issues are due to browser bugs. Note that the Chrome issue should be fixed in the latest version of Chrome, per bug 11. I am out of the office and haven't been able to test myself yet, though.

**Comment by yang3kui** [comment]
I have updated my chrome to ver: 44.0.2403.128 (released on 2015-07-31),
still not working.

I also tried chrome dev ver: 45.0.245412,  with no lucky.

---------------------------------------------------------
Kui Yang
Tel:    +86 18515060824

School of Instrumentation Science and Opto-electronics Engineering,
Beijing University of Aeronautics and Astronautics
No. 37, XueYuan Road, HaiDian District,
Beijing, 100191, China


2015-07-31 21:17 GMT+08:00 Andrea <notifications@github.com>:
